.travis.yml

```diff
@@ -13,34 +17,52 @@ jdk: oraclejdk8
 env:
   matrix:
     # ECLIPSE_TARGET is the targeted platform: mars is the default target
-    # JAVA_RUNTIME specifies the execution runtime for tests; must match an <id> in .travis-toolchains.xml
+    # JAVA_RUNTIME specifies the execution runtime for tests; must match an <id> in .travisci/toolchains.xml
     # SWT_GTK3=0 to force GTK2 on Mars as its GTK3 support is buggy
     - JAVA_RUNTIME=JavaSE-1.7 SWT_GTK3=0
     - JAVA_RUNTIME=JavaSE-1.8 ECLIPSE_TARGET=neon
     - JAVA_RUNTIME=JavaSE-1.8 ECLIPSE_TARGET=oxygen MAVEN_FLAGS=-Pjacoco
   global:
-    - PATH=$PWD/google-cloud-sdk/bin:$PATH
+    - CLOUD_SDK_HOME=$HOME/google-cloud-sdk
+    - PATH=$CLOUD_SDK_HOME/bin:$PATH
     # Our maven build typically reports about 512m
     - MAVEN_OPTS=-Xmx700m
     - CLOUDSDK_CORE_DISABLE_USAGE_REPORTING=true
     - DISPLAY=:99.0
+    - GCS_BUILD_BUCKET=gs://gcloud-eclipse-testing
 before_script:
   - "sh -e /etc/init.d/xvfb start"
   - sleep 3 # give xvfb some time to start
   - metacity --sm-disable --replace &
   - sleep 3 # give metacity some time to start
 script: ./mvnw -V -B -Ptravis --fail-at-end verify ${MAVEN_FLAGS}
      ${ECLIPSE_TARGET:+-Declipse.target=${ECLIPSE_TARGET}}
-     --toolchains=.travis-toolchains.xml
+     --toolchains=.travisci/toolchains.xml
      -Dtoolchain.java.runtime=${JAVA_RUNTIME}
-addons:
-  apt:
-    packages:
-      - metacity
 cache:
   directories:
    - $HOME/.m2
-   - $HOME/google-cloud-sdk
+   - $CLOUD_SDK_HOME
 after_success:
   # test req'd as we don't run coverage on all elements of the matrix
   - if [ -d build/jacoco/target ]; then bash <(curl -s https://codecov.io/bash); fi
+after_script:
+  # Upload build reports to our Google Cloud Storage bucket
+  - openssl aes-256-cbc -K $encrypted_42b011b17512_key -iv $encrypted_42b011b17512_iv
+      -in .travisci/travis-service-account.json.enc -out /tmp/travis-service-account.json -d
+  - $CLOUD_SDK_HOME/bin/gcloud auth activate-service-account --key-file=/tmp/travis-service-account.json
+  # Copy surefire-reports and screenshots
+  - mkdir build-results && .travisci/copy-test-results.sh plugins build-results
+  ###- $CLOUD_SDK_HOME/bin/gsutil -m -q
```

Comment:
Are these comments still relevant? Can we delete these lines?

Reply:
They're relevant: they're the final bits to upload the results.